### ISSUE TITLE
[BUG] Incorrect handling of ammo_id arrays

### ISSUE THREAD

**Issue by Quitch** [Quitch/GW-AI-Overhaul#8]
TITANS allowed for the use of an array of objects against a weapon's `ammo_id`. This is used by the torpedo launchers to differentiate their land and water ammos. The Galactic War referee cannot handle this, leading to the following errors in the log:

```
ERROR ammo_id json is incorrect. Array element is not an object
ERROR ammo_id json is incorrect. Object elements are not strings
```

This is due to it inserting the following into the weapon's spec:

`"a","/","a","m","m","o","/","t","o","r","p","e","d","o","/","t","o","r","p","e","d","o",".","j","s","o","n",".","p","l","a","y","e","r"`

The expectation is that `GW.specs.genUnitSpecs()` will need to be replaced by the mod, but the root cause has not yet been traced. Currently this appears to generate errors but not impact the application of mods to the unit,

**Comment by Quitch** [comment]
Confirmed that this error no longer occurs with the latest `genUnitSpecs()`

**Comment by Quitch** [comment]
This was actually fixed by commit 83af145
